core/input/input_event.h

```diff
@@ -289,6 +289,10 @@ class InputEventMouseMotion : public InputEventMouse {
 	Vector2 get_velocity() const;
 
 	virtual Ref<InputEvent> xformed_by(const Transform2D &p_xform, const Vector2 &p_local_ofs = Vector2()) const override;
+
+    virtual bool action_match(const Ref<InputEvent> &p_event, bool p_exact_match, float p_deadzone, bool *r_pressed, float *r_strength, float *r_raw_strength) const override;
```

Comment:
You need to use tabs for indentation, see the [code style](https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html)

Reply:
Thanks for the catch. I tried converting spaces to tabs when I could but my editor kept flipping back into spaces mode and I missed that one. I'll hit all the files just to be sure.